modules/odc-monitoring-your-app-vulnerabilities.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * applications/odc-monitoring-project-and-application-metrics-using-developer-perspective.adoc
+
+:_content-type: PROCEDURE
+[id="odc-monitoring-your-application-image-vulnerabilities-metrics_{context}"]
+= Monitoring your application and image vulnerabilities metrics
+
+After you create applications in your project and deploy them, use the *Developer* perspective in the web console to see the metrics for your application dependency vulnerabilities across your cluster. The metrics help you to analyze the following image vulnerabilities in detail:
+
+* Total count of vulnerable images in a selected project
+* Severity-based counts of all vulnerable images in a selected project
+* Drilldown into severity to obtain the details, such as count of vulnerabilities, count of fixable vulnerabilities, and number of affected pods for each vulnerable image
+
+.Prerequisites
+* You have installed the Red Hat Quay Container Security operator from the Operator Hub.
++
+[NOTE]
+====
+The Red Hat Quay Container Security operator detects the vulnerabilities by scanning the images that are in the quay registry.
+====
+
+.Procedure
+
+. For a general overview of the image vulnerabilities, on the navigation panel of the *Developer* perspective, click *Project* to see the project dashboard.
+
+. Click *Image Vulnerabilities* in the *Status* section. A pop-up opens containing the image vulnerabilities breakdown details, such as *Vulnerable Container Images* and *Fixable Container Images*.
```

Comment:
```suggestion
. Click *Image Vulnerabilities* in the *Status* section. The window that opens displays details such as *Vulnerable Container Images* and *Fixable Container Images*.
```
OR
```suggestion
. Click *Image Vulnerabilities* in the *Status* section to see details such as *Vulnerable Container Images* and *Fixable Container Images*.
```

- Use "window" instead of "pop-up."
- Apply minimalism.